### PR TITLE
3242 documentation

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do?

Adds documentation for the Malware Masquerade feature.
@ybasford 
## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] ~Was the CHANGELOG.md updated to reflect the changes?~
* [x] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working
masquerade

### CODE REVIEW

**Review by ilija-lazoroski — APPROVED** [review]

**Review by shreyamalviya — APPROVED** [review]

File: docs/content/usage/malware-masquerade.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: "Malware Masquerade"
+date: 2023-05-08T13:57:22-04:00
+draft: false
+weight: 4
+pre: "<i class='fas fa-mask'></i> "
+tags: ["usage"]
+---
+
+## Description
+**Masquerade** - to assume the appearance of something one is not
```
Comment (ybasford) [inline docs/content/usage/malware-masquerade.md:11]: **Masquerade** - assuming the appearance of something one is not.

File: docs/content/usage/malware-masquerade.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+[[1](https://www.merriam-webster.com/dictionary/masquerade)]
+
+A common way of detecting and identifying an executable as malware is to write
+a detection rule (such as for a tool like YARA) that checks for the existence
```
Comment (ybasford) [inline docs/content/usage/malware-masquerade.md:15]: a detection rule (as found in a tool like YARA) that checks for the existence

File: docs/content/usage/malware-masquerade.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+of strings or unique byte sequences within a file. The Malware Masquerade
+feature allows users to specify strings or arbitrary data (bytes) that will be
+injected into the Agent binaries. This enables Infection Monkey Agents to
+masquerade as specific types of malware. This is particularly useful for anyone
+who writes their own detection rules and needs a way to test them, or anyone
```
Comment (ybasford) [inline docs/content/usage/malware-masquerade.md:20]: writing their own detection rules and need a way to test them, or anyone

File: docs/content/usage/malware-masquerade.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+looking to improve the fidelity of malware simulations.
```
Comment (ybasford) [inline docs/content/usage/malware-masquerade.md:21]: looking to improve malware simulations' fidelity.

**Comment by mssalvatore** [comment]
Fixed in b3bf95fec

**Pull request closed without merging**